lib/srv/db/watcher_test.go

```diff
@@ -207,6 +207,15 @@ func TestWatcherDynamicResource(t *testing.T) {
 	})
 }
 
+func setDiscoveryGroupLabel(r types.ResourceWithLabels, discoveryGroup string) {
+	staticLabels := r.GetStaticLabels()
+	if staticLabels == nil {
+		staticLabels = make(map[string]string)
+	}
+	staticLabels[types.TeleportInternalDiscoveryGroupName] = discoveryGroup
```

Comment:
nit: In case of empty `discoveryGroup` we will add empty `discovery-group-name=` label